client/components/redux-forms/redux-form-textarea/index.jsx

```diff
@@ -3,39 +3,26 @@
  */
 import React, { PureComponent, PropTypes } from 'react';
 import { Field } from 'redux-form';
-import { omit } from 'lodash';
 
 /**
  * Internal dependencies
  */
 import FormTextarea from 'components/forms/form-textarea';
 
+// eslint-disable-next-line no-unused-vars
+const RenderTextarea = ( { input, meta, type, ...otherProps } ) => {
+	return (
+		<FormTextarea { ...input } { ...otherProps } ></FormTextarea>
+	);
+};
```

Comment:
Minor nitpick -- can we make this even terser? I.e. change `=> { return <SomeComp />; }` to `=>( <SomeComp /> )`? :slightly_smiling_face: